internal/dataplane/kong_client.go

```diff
@@ -451,14 +465,25 @@ func (c *KongClient) triggerKubernetesObjectReport(objs ...client.Object) {
 	// control-plane can update the Kubernetes object statuses for affected objs.
 	// this has to be done in a separate loop so that the filter is in place
 	// before the objects are enqueued, as the filter is used by the control-plane
-	for _, obj := range objs {
+	for _, obj := range reportedObjects {
+		c.logger.Infof("publishing event for object %s/%s/%s", obj.GetObjectKind().GroupVersionKind(), obj.GetNamespace(), obj.GetName())
```

Comment:
I think that might result in too many log entries to make it info level. WDYT about changing it to debug?

Reply:
This log is just for debugging and would be deleted later.

Reply:
removed the logs.